xml/depl_maintenance.xml

```diff
@@ -508,6 +508,262 @@
+       software on cluster nodes.
+      </para>
+     </step>
+     <step>
+      <para>
+       To determine whether the &postgres; databases contain data that cannot
+       be migrated to &mariadb;, run <command>psql2mysql</command> with the
+      <literal>precheck</literal> option:</para>
+<screen>psql2mysql \
+--source postgresql://keystone:vZn3nfxXzv97@192.168.243.87/keystone \
+--target mysql+pymysql://keystone:vZn3nfxXzv97@192.168.243.88/keystone?charset=utf8 \
+precheck</screen>
+      <para>
+       To run precheck operation on all databases in a single operation, use the
+       <literal>--batch</literal> option and the
+       <filename>/etc/pg2mysql/databases.yaml</filename> file as follows:
+      </para>
+<screen>psql2mysql --batch /etc/pg2mysql/databases.yaml precheck</screen>
+      <para>
+       If the precheck indicates that there is data that cannot be imported
+       into &mariadb;, modify the offending data manually to fix
+       the problems.
+      </para>
+     </step>
+     <step>
+      <para>
+       Stop all &ostack; services that make use of the database to prevent them
+       from writing new data during the migration. Stop shef-client services on
+       the nodes, to prevent regular runs from starting database-related
+       &ostack; services again. <emphasis>From this point, &ostack; services
+       become unavailable.</emphasis>
+      </para>
+     </step>
+     <step>
+      <para>
+       You can now migrate databases using the psql2mysql tool. To upgrade a
+       single database, use the following command format:
+      </para>
+<screen>psql2mysql \
+--source postgresql://neutron:secret@192.168.1.1/neutron \
+--target mysql+pymysql://neutron:evenmoresecret@192.168.1.2/neutron?charset=utf8 \
+migrate</screen>
+      <para>
+       To migrate all databases in one operation, use the
+       <literal>--batch</literal> option and the
+       <filename>/etc/pg2mysql/databases.yaml</filename> file as follows:
+      </para>
+<screen>psql2mysql.py --batch /etc/pg2mysql/databases.yaml migrate</screen>
+     </step>
+     <step>
+      <para>
+       In the &crow; &wi;, switch to the <guimenu>Database</guimenu>
+       &barcl;. Enable the raw view and set the value of
+       <literal>sql_engine</literal> to <literal>mysql</literal>. Apply the
+       &barcl;. After this step, &ostack; services should be running again and
+       reconfigured to use the &mariadb; database back end.
+      </para>
+     </step>
+     <step>
+      <para>
+        To prevent the &postgres;-related chef code from running, unassign the values
+	from <literal>database-server</literal> role in the <guimenu>Database</guimenu>
+       &barcl;, and apply the &barcl;.
+      </para>
+     </step>
+     <step>
+      <para>
+       Start chef-client services on the nodes again.
+      </para>
+     </step>
+     <step>
+      <para>
+       Stop &postgres; on the &dbnode;s. Uninstall &postgres;packages.
```

Comment:
There's a space missing between "&postgres;" and packages

Reply:
Fixed.